services/federator/src/Federator/Options.hs

```diff
@@ -92,7 +92,9 @@ data Opts = Opts
     -- | Logformat to use
     logFormat :: !(Maybe (Last LogFormat)),
     -- | Runtime settings
-    optSettings :: !RunSettings
+    optSettings :: !RunSettings,
+    -- | Domain update interval (microseconds)
+    domainUpdateInterval :: !Int
```

Comment:
only make this configurable in brig, and serve the parameter together with the remotes in the rest api?